test/unit/authorization/WebAclAuthorizer.test.ts

```diff
@@ -155,4 +155,20 @@ describe('A WebAclAuthorizer', (): void => {
     await expect(promise).rejects.toThrow('No ACL document found for root container');
     await expect(promise).rejects.toThrow(InternalServerError);
   });
+
+  it('allows an agent to append if they have write access.', async(): Promise<void> => {
+    credentials.webId = 'http://test.com/user';
+    identifier.path = 'http://test.com/foo';
+    permissions = {
```

Comment:
In unit tests (and elsewhere), we might want to distinguish between `requiredPermissions` and `actualPermissions`.

Reply:
What would be the difference? Because the only permissions object that I can think of that is being passed around are those that are required for the request.

Reply:
So when I read this test, I go:

1. The user has WebID `http://test.com/user`
2. The user requests path `http://test.com/foo`
3. The user has permissions: append

But then I have to backtrack because the last thing is not the permissions the user has, but rather the permissions to user needs. So naming it `requiredPermissions` would avoid that misreading.